be/src/runtime/tablets_channel.cpp

```diff
@@ -281,9 +292,7 @@ Status TabletsChannel::_open_all_writers(const PTabletWriterOpenRequest& request
         }
     }
     if (index_slots == nullptr) {
-        std::stringstream ss;
-        ss << "unknown index id, key=" << _key;
-        return Status::InternalError(ss.str());
+        return Status::InternalError("unknown index id, key=" + _key.to_string());
```

Comment:
Recommend to use `Status::InternalError("unknown index id, key={}", _key.to_string());`